app/Jobs/AdsSendOne.php

```diff
@@ -81,7 +81,7 @@ public function handle(AdsClient $adsClient)
         if (AdsValidator::isTransactionIdValid($txid)) {
             // TODO move to queue (or service in general) to assure user ledger entry update
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1135)